wee_slack.py

```diff
@@ -3615,6 +3659,9 @@ def get_string(self, key):
     def get_int(self, key):
         return int(w.config_get_plugin(key))
 
+    def is_default(self, key):
+        return w.config_get_plugin(key) == self.default_settings.get(key)
```

Comment:
`w.config_get_plugin` returns a string while `default_settings.get` returns a `Settings` object, so this comparison doesn't work.